pkg/sessionctx/variable/sysvar.go

```diff
@@ -453,6 +454,19 @@ var defaultSysVars = []*SysVar{
 	}, GetGlobal: func(_ context.Context, s *SessionVars) (string, error) {
 		return BoolToOnOff(vardef.ProcessGeneralLog.Load()), nil
 	}},
+	// NOTE: The trace-event switch is experimental. It is subject to changes.
+	{Scope: vardef.ScopeGlobal, Name: vardef.TiDBTraceEvent, Value: vardef.DefTiDBTraceEvent, Type: vardef.TypeStr,
+		SetGlobal: func(_ context.Context, _ *SessionVars, val string) error {
+			if kerneltype.IsClassic() {
```

Comment:
Should we hide it for classic version as conditional compile is not supported in tidb?